sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -318,7 +318,7 @@ query
 
 insertInto
     : INSERT OVERWRITE TABLE? identifierReference (partitionSpec (IF NOT EXISTS)?)?  identifierList?         #insertOverwriteTable
-    | INSERT INTO TABLE? identifierReference partitionSpec? (IF NOT EXISTS)? identifierList?                 #insertIntoTable
+    | INSERT INTO TABLE? identifierReference partitionSpec? (BY NAME)? (IF NOT EXISTS)? identifierList?                 #insertIntoTable
```

Comment:
shall we do`((BY NAME) | identifierList)`?

Reply:
Done